content/_code-samples/get-started/py/prepare-payment.py

```diff
@@ -7,7 +7,7 @@
 
 # Create a wallet using the testnet faucet:
 # https://xrpl.org/xrp-testnet-faucet.html
-from xrpl.wallet import generate_faucet_wallet
+# from xrpl.wallet import generate_faucet_wallet
```

Comment:
I think there's some mistake here. If you comment out this import, the code sample no longer works. I'm not sure why this PR should be changing a code sample anyway.